kii-core/kii_core.c

```diff
@@ -671,11 +672,19 @@ prv_http_request(
         const char* content_type,
         const char* access_token,
         const char* etag,
-        const char* body)
+        const char* body,
+        const char* encoding)
```

Comment:
`content_encoding`

There's several `encoding` param in HTTP.
ex.) Transfer-Encoding